internal/hostport/noop_hostport_manager.go

```diff
@@ -0,0 +1,21 @@
+package hostport
+
+import "github.com/sirupsen/logrus"
+
+type noOpHostportManager struct{}
+
+// NewNoOpHostportManager creates a new HostPortManager
+func NewNoOpHostportManager() HostPortManager {
+	logrus.Info("HostPort Mapping is Disabled in CRI-O")
+	return &noOpHostportManager{}
+}
+
+func (mh *noOpHostportManager) Add(id string, podPortMapping *PodPortMapping, natInterfaceName string) error {
+	logrus.Info("HostPort Mapping is Disabled in CRI-O")
```

Comment:
these infos seem a bit excessive, as they'll be called each new container. I think the single one when the noop manager is created is sufficient. If anything, maybe they should be dropped to debug level?